src/theory/rewriter.cpp

```diff
@@ -29,7 +30,7 @@ namespace CVC4 {
 namespace theory {
 
 static TheoryId theoryOf(TNode node) {
-  return Theory::theoryOf(THEORY_OF_TYPE_BASED, node);
+  return Theory::theoryOf(options::TheoryOfMode::THEORY_OF_TYPE_BASED, node);
```

Comment:
FYI you'll have a merge conflict here (this code does not depend on options::TheoryOfMode::THEORY_OF_TYPE_BASED anymore, keep the code in master unchanged).

Reply:
Done, thanks for the info!